packages/turbo-codemod/src/commands/migrate/steps/getLatestVersion.ts

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+
+import { MigrateCommandOptions } from "../types";
+
+const REGISTRY = "https://registry.npmjs.org";
+
+async function getLatestVersion({
+  to,
+}: MigrateCommandOptions): Promise<string | undefined> {
+  if (to) {
+    return Promise.resolve(to);
+  }
+
+  try {
+    const result = await axios.get(`${REGISTRY}/turbo`);
```

Comment:
Can we use fetch? (or undici for older Node support)

Reply:
Fetch isn't native to node (_yet_, it's partially in 18, but we can't rely on that). So we would need a dependency here regardless, and axios is the defacto standard for node (we also use axios in docs and turbopack already).

Reply:
Agreed, we need a dependency, but we should we looking to move towards undici, since it's effectively the polyfill for spec-compliant fetch in Node. (It _is_ the implementation pulled into node itself).